maven-plugin-report-plugin/src/it/fix-maven-since-3.x/pom.xml

```diff
@@ -81,7 +81,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-help-plugin</artifactId>
-        <version>2.1.1</version>
+        <version>3.4.0</version>
```

Comment:
property?

Reply:
This is single use, here too, on purpose none